### PR TITLE
feat: change font with variable Roboto Flex font

### PR DESCRIPTION
**Pull request by yilmazcabuk**
With the development of the modern web, old static fonts are now being replaced by variable fonts. As we can see in [this](https://material.io/blog/roboto-flex) article from Google, the variable version of the Roboto font appears as Roboto Flex.

### CODE REVIEW

File: src/common/AddonDetailsModal/AddonDetails/styles.less [hunk 1]
```diff
@@ -8,7 +8,8 @@
         flex-direction: row;
         align-items: center;
 
-        .logo, .icon {
```
Comment (nklhtv) [inline src/common/AddonDetailsModal/AddonDetails/styles.less:11]: Those changes must be reverted and sent in a separate PR

File: src/common/CONSTANTS.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 const CHROMECAST_RECEIVER_APP_ID = '1634F54B';
 const SUBTITLES_SIZES = [75, 100, 125, 150, 175, 200, 250];
-const SUBTITLES_FONTS = ['Roboto', 'Arial', 'Halvetica', 'Times New Roman', 'Verdana', 'Courier', 'Lucida Console', 'sans-serif', 'serif', 'monospace'];
+const SUBTITLES_FONTS = ['Roboto Flex', 'Roboto', 'Arial', 'Halvetica', 'Times New Roman', 'Verdana', 'Courier', 'Lucida Console', 'sans-serif', 'serif', 'monospace'];
```
Comment (nklhtv) [inline src/common/CONSTANTS.js:5]: Should we keep 'Roboto' here or replace it with 'Roboto Flex' instead?

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,4 @@
 /yarn.lock
 /npm-debug.log
 .DS_Store
+compose-dev.yaml
```
Comment (nklhtv) [inline .gitignore:6]: Why `compose-dev.yaml` is added to .gitignore

**Review by nklhtv — CHANGES_REQUESTED** [review]

File: src/common/CONSTANTS.js (hunk 2 above)
Comment (yilmazcabuk) [reply src/common/CONSTANTS.js:5]: In my opinion we should keep 'Roboto' as a fallback option.
